### PR TITLE
dev-libs/igraph: version bump to 0.8.3

### PR DESCRIPTION
**Pull request by epsilon-0**
Package-Manager: Portage-3.0.8, Repoman-3.0.1
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: dev-libs/igraph/igraph-0.8.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+HOMEPAGE="http://www.igraph.org/"
+SRC_URI="https://github.com/igraph/igraph/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0/0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND="
+	dev-libs/gmp:0
```
Comment (SoapGentoo) [inline dev-libs/igraph/igraph-0.8.3.ebuild:18]: `:0=`

File: dev-libs/igraph/igraph-0.8.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	dev-libs/libxml2
+	sci-libs/arpack
+	sci-libs/cxsparse
+	sci-mathematics/glpk
```
Comment (SoapGentoo) [inline dev-libs/igraph/igraph-0.8.3.ebuild:22]: `:=`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-libs/igraph/igraph-0.8.3.ebuild (hunk 1 above)
Comment (epsilon-0) [reply dev-libs/igraph/igraph-0.8.3.ebuild:18]: done

File: dev-libs/igraph/igraph-0.8.3.ebuild (hunk 2 above)
Comment (epsilon-0) [reply dev-libs/igraph/igraph-0.8.3.ebuild]: done

**Review by SoapGentoo — APPROVED** [review]

**Pull request closed without merging**